src/tabs/heatmap/column-filters/ColumnFiltersSection.js

```diff
@@ -12,6 +12,10 @@ const headerName = (name) => (
   + `: `
 )
 
+const humanize = (name)=> (
+	name.split(`_`).join(` `).toLowerCase()
+)
+
```

Comment:
For the record, if `name` is the only argument in the function and the result is the return value of `toLowerCase` you can simplify this to:
```javascript
const humanize = name => name.split(`_`).join(` `).toLowerCase()
```

And going one step further, notice that this `split` and `join` is equivalent to:
```javascript
const humanize = name => name.replaceAll(`_`, ` `).toLowerCase()
```

Reply:
Replaced with above. Thanks!

Reply:
Do I need to publish this change